src/OSPSuite.Core/Domain/PopulationSimulationPKAnalyses.cs

```diff
@@ -46,7 +46,7 @@ public virtual bool HasPKParameterFor(string quantityPath, string pkParameter)
       /// </summary>
       public virtual IReadOnlyList<QuantityPKParameter> AllPKParametersFor(string quantityPath)
       {
-         return _pkAnalyses.Where(x => string.Equals(x.QuantityPath, quantityPath)).ToList();
+         return _pkAnalyses.Where(x => string.Equals(x.QuantityPath, quantityPath)).ToArray();
```

Comment:
R does not know how to work with list.
Based on the returned value (IReadOnlyList) it should be equivalent to return a ToArray